### PR TITLE
Test new bigdl-nano-init

### PR DESCRIPTION
**Pull request by az2181036**

### CODE REVIEW

File: python/nano/scripts/bigdl-nano-init [hunk 1]
```diff
@@ -215,11 +215,20 @@ if [ ! -z "${DISABLE_TCMALLOC_VAR:-}" ]; then
...
+	echo "Not in a conda env"
+else
+    if [ -f $CONDA_PREFIX/etc/conda/activate.d/nano_vars.sh ];then
+        echo "nano_vars.sh already exists"
+    else
+        echo "Setting environment variables in current conda env"
+        mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+        mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d
+        echo "source bigdl-nano-init" > $CONDA_PREFIX/etc/conda/activate.d/nano_vars.sh
+		echo "source ${CONDA_PREFIX}/bin/bigdl-nano-unset-env" > $CONDA_PREFIX/etc/conda/deactivate.d/nano_vars.sh
```
Comment (yangw1234) [inline python/nano/scripts/bigdl-nano-init:228]: fix this indentation?

**Comment by yangw1234** [comment]
LGTM except one style issue

File: python/nano/scripts/bigdl-nano-init (hunk 1 above)
Comment (az2181036) [reply python/nano/scripts/bigdl-nano-init:228]: Yes, it seems scripts in the deactivate.d folder run after exit the environment, so I add path prefix of `bigdl-nano-unset-env`.

File: python/nano/scripts/bigdl-nano-init [hunk 2]
```diff
@@ -215,11 +215,29 @@ if [ ! -z "${DISABLE_TCMALLOC_VAR:-}" ]; then
...
+        mkdir -p $DEACTIVATED_PATH
+        # bigdl-nano-init
+        echo "if [-d ${CONDA_PREFIX}/bin/bigdl-nano-init ]; then}" > $ACTIVATED_PATH/nano_vars.sh
+        echo "    source bigdl-nano-init" >> $ACTIVATED_PATH/nano_vars.sh
+        echo "fi" >> $ACTIVATED_PATH/nano_vars.sh
+
+        #bigdl-nano-unset-env
+        echo "if [-d ${CONDA_PREFIX}/bin/bigdl-nano-unset-env ]; then}" > $DEACTIVATED_PATH/nano_vars.sh
+        echo "    source ${CONDA_PREFIX}/bin/bigdl-nano-unset-env" >> $DEACTIVATED_PATH/nano_vars.sh
+        echo "fi" >> $DEACTIVATED_PATH/nano_vars.sh
```
Comment (yangw1234) [inline python/nano/scripts/bigdl-nano-init:237]: 1. maybe also add
```
echo "Added nano_vars.sh script to $ACTIVATED_PATH and $DEACTIVATED_PATH. You may want to delete them if you want to uninstall bigdl-nano"
```

2. Inside nano_vars.sh
if we cannot find bigdl-nano-init and bigdl-nano-unset-env, maybe we can print 
```
echo "cannot find bigdl-nano-init, if you have uninstalled bigdl-nano, you may want to delete $ACTIVATED_PATH/nano_vars.sh and $DEACTIVATED_PATH/nano_vars.sh"
```

**Comment by az2181036** [comment]
http://10.112.231.51:18888/job/ZOO-PR-NanoTests/405/

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/ZOO-PR-NanoTests/413/

**Pull request merged**